diff --git a/ckan/logic/action/get.py b/ckan/logic/action/get.py
index 7ee7438..fa348e6 100644
--- a/ckan/logic/action/get.py
+++ b/ckan/logic/action/get.py
@@ -551,6 +551,7 @@ def group_show(context, data_dict):
 
     if schema:
         package_dict, errors = validate(group_dict, schema, context=context)
+        group_dict, errors = validate(group_dict, schema, context=context)
 
     return group_dict
 
