diff --git a/ckan/controllers/group.py b/ckan/controllers/group.py
index 21c5bab..f282e26 100644
--- a/ckan/controllers/group.py
+++ b/ckan/controllers/group.py
@@ -31,7 +31,7 @@ class GroupController(BaseController):
     def _db_to_form_schema(self, group_type=None):
         '''This is an interface to manipulate data from the database
         into a format suitable for the form (optional)'''
-        return lookup_group_plugin(group_type).form_to_db_schema()
+        return lookup_group_plugin(group_type).db_to_form_schema()
 
     def _setup_template_variables(self, context, data_dict, group_type=None):
         return lookup_group_plugin(group_type).setup_template_variables(context,data_dict)
@@ -99,7 +99,7 @@ class GroupController(BaseController):
         context = {'model': model, 'session': model.Session,
                    'user': c.user or c.author,
                    'schema': self._form_to_db_schema(group_type=group_type),
-                   'for_view': True}
+                   'for_view': True, 'extras_as_string': True}
         data_dict = {'id': id}
         q = c.q = request.params.get('q', '') # unicode format (decoded from utf8)
 
@@ -175,7 +175,6 @@ class GroupController(BaseController):
             fq = 'capacity:"public"'
             if (c.userobj and c.group and c.userobj.is_in_group(c.group)):
                 fq = ''
-                context['ignore_capacity_check'] = True
 
             data_dict = {
                 'q':q,
