diff --git a/ckan/controllers/home.py b/ckan/controllers/home.py
index d1621d6..f307cb7 100644
--- a/ckan/controllers/home.py
+++ b/ckan/controllers/home.py
@@ -41,19 +41,22 @@ class HomeController(BaseController):
             data_dict = {
                 'q':'*:*',
                 'facet.field':g.facets,
-                'rows':0,
+                'rows':4,
                 'start':0,
-                'fq': 'capacity:"public"'
+		'sort': 'views_recent desc',
             }
+	    context['ignore_capacity_check'] = True
             query = ckan.logic.get_action('package_search')(context,data_dict)
             c.package_count = query['count']
+	    c.datasets = query['results']
             c.facets = query['facets']
 
             data_dict = {'order_by': 'packages', 'all_fields': 1}
+	    c.search_facets = query['search_facets']
             #only give the terms to group dictize that are returned in the facets
             #as full results take a lot longer
-            if 'groups' in c.facets:
-                data_dict['groups'] = c.facets['groups'].keys()
+	    if 'groups' in c.search_facets:
+		data_dict['groups'] = [ item['name'] for item in c.search_facets['groups']['items'] ]
             c.groups = ckan.logic.get_action('group_list')(context, data_dict)
         except SearchError, se:
             c.package_count = 0
