diff --git a/ckan/lib/i18n.py b/ckan/lib/i18n.py
index 87e7312..346468e 100644
--- a/ckan/lib/i18n.py
+++ b/ckan/lib/i18n.py
@@ -91,7 +91,7 @@ def handle_request(request, tmpl_context):
     ''' Set the language for the request '''
     lang = request.environ.get('CKAN_LANG') or \
                         config.get('ckan.locale_default', 'en')
-    if lang != 'en':
+    if 1 or lang != 'en':
         i18n.set_lang(lang)
     tmpl_context.language = lang
     return lang
